syntax = "proto3";
import "graph.proto";

package TCProto;

message Item {
    uint32 edge_id = 1;
    uint32 index_bus = 2;
	uint32 span_count = 3;
};

message BusGraph {
    uint32 bus_wait_time = 1;
    uint32 bus_velocity = 2;
    repeated Item items = 3;
};
